elastic/datadog_checks/elastic/elastic.py

```diff
@@ -48,79 +48,79 @@ def __init__(self, name, init_config, instances):
                 'name': 'aws_host',
                 'default': urlparse(self.instance['url']).hostname,
             }
+        self.config = from_instance(self.instance)
 
-    def check(self, instance):
-        config = from_instance(instance)
-        admin_forwarder = config.admin_forwarder
-        jvm_rate = instance.get('gc_collectors_as_rate', False)
+    def check(self, _):
+        admin_forwarder = self.config.admin_forwarder
+        jvm_rate = self.instance.get('gc_collectors_as_rate', False)
 
         # Check ES version for this instance and define parameters
         # (URLs and metrics) accordingly
         try:
-            version = self._get_es_version(config)
+            version = self._get_es_version()
         except AuthenticationError:
             self.log.exception("The ElasticSearch credentials are incorrect")
             raise
 
-        health_url, stats_url, pshard_stats_url, pending_tasks_url = self._get_urls(version, config.cluster_stats)
+        health_url, stats_url, pshard_stats_url, pending_tasks_url = self._get_urls(version, self.config.cluster_stats)
```

Comment:
```suggestion
        health_url, stats_url, pshard_stats_url, pending_tasks_url = self._get_urls(version)
```

Small nit but `config` is still being passed in here